guidelines.md

```diff
@@ -77,4 +125,24 @@ You can utilize our existing SDKs functions to handle access token
 
 Stuck somewhere in implementation, Raise a support ticket. We'll be happy to help you.
 
-![Image](support_ticket.png)
\ No newline at end of file
+![Image](support_ticket.png)
+
+
+
+## FAQs
+
+1. How we are reading the users Access Token?
+
+After successful login from LoginRadius IDX, the access token is returned as the query parameter to the callback URL, Access token is been read from the callback URL and stored in the browser.
```

Comment:
`Access token is been read from the callback URL` can be `Access token is read from the callback URL`